dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/exchange/codec/ExchangeCodec.java

```diff
@@ -152,7 +152,8 @@ protected Object decodeBody(Channel channel, InputStream is, byte[] header) thro
                 if (status == Response.OK) {
                     Object data;
                     if (res.isHeartbeat()) {
-                        data = decodeHeartbeatData(channel, in);
+                        // data = decodeHeartbeatData(channel, in);

```

Comment:
would it be better to implement it like this?

```java
    @Deprecated
    protected Object decodeHeartbeatData(Channel channel, ObjectInput in) throws IOException {
        return Response.HEARTBEAT_EVENT;
    }
```

Reply:
Yes, it's better to implement.